src/module.c

```diff
@@ -997,16 +997,9 @@ int RediSearch_InitModuleInternal(RedisModuleCtx *ctx, RedisModuleString **argv,
 
   RM_TRY(NumericIndexType_Register, ctx);
 
-#ifndef RS_COORDINATOR
-// on a none coordinator version (for RS light/lite) we want to raise cross slot if
-// the index and the document do not go to the same shard
-#define INDEX_ONLY_CMD_ARGS 1, 1, 1
-#define INDEX_DOC_CMD_ARGS 1, 2, 1
```

Comment:
I think this is still needed for `LITE`, without it commands won't get redirected to the correct shard